packages/shared-ux/button/exit_full_screen/src/fallback.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import { useEuiTheme, EuiButton } from '@elastic/eui';
+
+const loading = i18n.translate('sharedUXPackages.exitFullScreenButton.loadingLabel', {
+  defaultMessage: 'Loading',
+});
+
+export const Fallback = () => {
+  const { euiTheme } = useEuiTheme();
+  const { size, border } = euiTheme;
+
+  const fallbackCSS = css`
+    padding: ${size.xs} ${size.s};
+    border-radius: ${border.radius.small};
+    height: ${size.xl};
+  `;
+
+  return <EuiButton css={fallbackCSS} isLoading={true} aria-label={loading} />;
```

Comment:
I think this is going to produce a pretty major flash/component swap between the loading version and the final button. This button will be our basic blue, lightly shaded button and the final full screen one is a completely custom version with dark background. To me this is the one place I'm not sure we need any loading swap at all.

Reply:
Yeah, I was thinking this last night-- just use an empty `<span />` instead?

Reply:
Yeah if it needs to render **something**, that would be fine